docs/faq.md

```diff
@@ -1,5 +1,13 @@
 # Frequently Asked Questions
 
+### Something is wrong with my config, how can I find where the problem comes from?
+
+`vite` and `vite-plugin-svelte` can be made more verbose by using the debug mode:
+
+- `export DEBUG=vite-plugin-svelte:config ; vite`: log `vite-plugin-svelte` debug info related to the config
+- `export DEBUG=vite-plugin-svelte:* ; vite`: log all `vite-plugin-svelte` debug info
+- `export DEBUG=* ; vite`: log all debug info
```

Comment:
```suggestion
- `DEBUG=vite-plugin-svelte:config vite dev`: log `vite-plugin-svelte` debug info related to the config
- `DEBUG=vite-plugin-svelte:* vite dev`: log all `vite-plugin-svelte` debug info
- `DEBUG=* vite dev`: log all debug info
```

does export work on windows as well? I usually prefer putting it inline. Maybe a link to the documentation of the debug package can help too.

Reply:
Suggestions commited. For the link to `debug`, something like 1ea50866?